CHANGELOG

```diff
@@ -1,3 +1,72 @@
+== OpenDataPlane (1.19.0.0)
+=== Summary of Changes
+ODP v1.19.0.0 is the official Tiger Moth final release. It incorporates final
+implementation changes and bug fixes and completes the Tiger Moth ODP
+development cycle.
+
+==== APIs
+No changes for this release. The Tiger Moth API was frozen in ODP v1.18.0.0.
+
+==== C++ Test Improvements
+The $$C++$$ test included in the ODP validation suite now uses `cout` instead
+of `printf()` to ensure that $$C++$$ is being used to compile it.
+
+==== Queue and Scheduler Configuration
+For the ODP Reference Implementation, The `config/odp-linux-generic.conf` file
+is extended with sections to control the default and maximum sizes for basic
+queues, and the priority spread used by the scheduler for scheduled queues.
+
+==== Runtime Default `config` File Read Order Improvements
+For the ODP Reference Implementation, if an application-provided `config` file
```

Comment:
"For the ODP Reference Implementation, the default values of application-provided `config` file (if used) overrides the values provided by the built-in `config/odp-linux-config.conf` file."